packages/webview_flutter/README.md

```diff
@@ -0,0 +1,48 @@
+
+# webview_flutter_tizen
```

Comment:
Does it only work on arm targets, not x86(emulator)? If so, could you add some guide?

Reply:
This is a commit to work on the dev branch. The parts you mentioned will be worked on before entering the master.

Reply:
Since it is still in the early stages of development, there are many areas to be decided.(Including what you mentioned above)

Reply:
BTW, how did you know this PR came up? Do you get notified by email or something :question:

Reply:
Yes, Gmail forwards notices to Knox mail